pushy/src/main/java/com/eatthepath/pushy/apns/util/LiveActivityEvent.java

```diff
@@ -0,0 +1,17 @@
+package com.eatthepath.pushy.apns.util;
+
+public enum LiveActivityEvent {
```

Comment:
I'm cool with enumerating these, but I do predict that, since this is a new feature, there will be some iteration on what's allowed in here in the not-too-distant future. Having this in an enum means that we need to put out a new release of Pushy if Apple ever changes the allowed values.

We _could_ think about doing what we do with `ApnsPayloadBuilder#DEFAULT_SOUND_FILENAME` and just making these string constants (`LIVE_ACTIVITY_EVENT_UPDATE` and `LIVE_ACTIVITY_EVENT_END`) for example, then letting callers set the event type as a string. That would make things a little more flexible with all the good and bad that comes with flexibility.

On balance, though, I think enumerating is probably the right move. That said, if we're introducing a new enum, I think it needs some Javadoc.